src/CountUp.js

```diff
@@ -200,4 +195,95 @@ class CountUp extends Component {
   }
 }
 
+const NO_ELEMENT = -1;
+
+const decorator = (f, action) => (...args) => {
+  const result = f(...args);
+  action(result);
+  return result;
+};
```

Comment:
Since that's not used often, let's skip this and just call the original fn? Also easier to read imho